.firebaserc

```diff
@@ -21,10 +21,11 @@
     "unesco": "sparkle-unesco",
     "ohbm": "sparkle-ohbm",
     "sparkle-pa": "sparkle-pa",
-    "sparkle-demo": "sparkle-demo-e65b1",
+    "sparkle-demo-e65b1": "sparkle-demo-e65b1",
```

Comment:
I wonder if you can leave it as `sparkle-demo` here. Not sure about all of the consequences though.